docs/ui/project-proposal.md

```diff
@@ -0,0 +1,34 @@
+# New Repository Proposal: ui
+
+## Summary
+
+This document proposes a new repository under the `instructlab` GitHub organization:
+
+- `instructlab/ui`
+
+## Background
+
+Currently we don’t have public repository under InstructLab that hosts UI-related work. The goal of this project is to provide a space for hacking a UI for the InstructLab. The scope of this project is to support the following personas through the UI:
+
+- External contributors who want to contribute skill and knowledge to the taxonomy repo
+- Triagers who will be evaluating the taxonomy contributions.
```

Comment:
Could you clarify if you would only focus on the upstream project workflow, or if you would be creating something people could use to help with managing their own taxonomy repos?  Is it both?

Reply:
@russellb Main focus of this project is for the upstream workflow (for upstream contributors and upstream traigers). Intent is to develop it as a SaaS service and deploy it somewhere on Cloud (ROSA, IBM Cloud). But that doesn't stop users from deploying this stack in their private data center to manage their own private taxonomy repo if they want to follow similar workflow downstream.

Reply:
@russellb Let me know if you have any follow-up questions, otherwise I will push the patch to update the doc.

Reply:
I think something designed to be reused is important, too, so I just want to make it really clear whether this project is aiming to serve that need for instructlab or if it's explicitly an upstream-oriented SaaS only.

I think you're saying

- build something deployed as a SaaS for the upstream project and its taxonomy repo and process
- build in a way that someone could deploy the same service on their own infrastructure for managing their own taxonomy repo

Is that right?

Reply:
Yes, thats correct.

Reply:
Done. Thanks for the thorough review @russellb